lib/source-code/source-code.js

```diff
@@ -588,6 +588,31 @@ class SourceCode extends TokenStore {
 
         return positionIndex;
     }
+
+    /**
+     * Gets the scope for the given node
+     * @param {ASTNode} currentNode The node to get the scope of
+     * @returns {eslint-scope.Scope} The scope information for this node
+     */
+    getScope(currentNode) {
```

Comment:
Maybe this function should throw an error if called without the `currentNode` argument? I imagine that will be a common mistake when switching from `context.getScope()` to `sourceCode.getScope(node)` in plugins.